db/schema.rb

```diff
@@ -48,7 +48,7 @@
     t.index ["primary_phone_id"], name: "index_core_contacts_on_primary_phone_id"
   end
 
-  create_table "core_emails", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+  create_table "core_emails", force: :cascade do |t|
```

Comment:
Yeah no uuid here. While we don't edit this file directly I'm wondering why it's like this. Again drop the db, recreate, migrate and then annotate again let's see if that fixes it.